content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -49,6 +56,11 @@ npm install typescript \
   @types/express
 ```
 
+```sh
+#initialise typescript
+npx tsc --init
+```
```

Comment:
You could merge this code block with the one above

Reply:
This has not been resolved, please only close conversations if addressed or by providing a reason why you are not addressing them

Reply:
> This has not been resolved, please only close conversations if addressed or by providing a reason why you are not addressing them

How do I merge the changes? Im relatively new to pull request and havent grasped the concept of resolving the suggestions

Reply:
in that particular case you will need to fix things locally, open up that file again and merge the two code blocks into one, push the commit and let me know about the change

Reply:
I have commited and pushed the changes  @svrnm ...kindly let me know if everything is okay

Reply:
looks good now, thanks!